ctapipe/image/tests/test_extractor.py

```diff
@@ -112,6 +112,13 @@ def test_extract_pulse_time_around_peak(camera_waveforms):
     assert_allclose(pulse_time[0], 41.2, rtol=1e-3)
 
 
+def test_extract_pulse_time_around_peak_within_range(example_event):
+    telid = list(example_event.r0.tel)[0]
+    waveforms = example_event.r1.tel[telid].waveform
```

Comment:
it may be nicer to define some  dummy waveforms here, where you know the real peak, so this doesn't change when we update the test event file. That way you can also check that you reconstruct the "real" peak